student.cpp

```diff
@@ -106,13 +106,15 @@ void Student::setGradeableItemGrade_AcademicIntegrity(GRADEABLE_ENUM g, int i, f
   itr->second[i] = ItemGrade(value,late_days_used,note,status,temp,academic_integrity);
 }
 
-void Student::setGradeableItemGrade_border(GRADEABLE_ENUM g, int i, float value, const std::string &event, int late_days_used, const std::string &note, const std::string &status) {
+void Student::setGradeableItemGrade_border(GRADEABLE_ENUM g, int i, float value, const std::string &event, 
+                                           int late_days_used, const std::string &note, const std::string &status, int exceptions, const std::string &reason) {
   assert (i >= 0 && i < GRADEABLES[g].getCount());
   std::map<GRADEABLE_ENUM,std::vector<ItemGrade> >::iterator itr = all_item_grades.find(g);
   assert (itr != all_item_grades.end());
   assert (int(itr->second.size()) > i);
+  bool temp = false;
```

Comment:
what is this temp boolean for?

Reply:
It's a placeholder value used to initialize the academic integrity status of the ItemGrade object constructed to false.

Reply:
I just changed the name to bool academic_integrity to make it clearer.

Reply:
you are right, I think I did that for academic integrity as well